test_dir/new_file_1

```diff
@@ -0,0 +1,4 @@
+lets see, here is the file
+well heres what
+what now
+what now, well see
```

Comment:
RiflerRick committed on a restricted directory: test_dir/new_file_1in the repository: BB_MCS_Test belonging to Rajdeep Mukherjee(RiflerRick) authored by @RiflerRick in the file test_dir/new_file_1
Collaborators: @manasisingh67, @RiflerRick,